API.md

```diff
@@ -41,7 +41,7 @@ Request Response:
 * The top two parameters are derived by the KNN plugin from the customer's repo setting. `container_name` is used to specifically refer to the "bucket" (or non-S3 equivalent), rather than the name of the repository itself.
 * Tenant ID is included to be used for billing, authorization, etc
 * Dimension, doc count, and data type are specifically placed on the first JSON level so the build service can quickly use them first to calculate workload size
-* Doc count must be greater than 4, since doc counts of 1, 3, and 4 have been observed to consistently fail during index builds
+* Doc count must be greater than 4, since doc counts of 1, 3, and 4 have been observed to consistently fail during index builds. There is currently an issue open with faiss for this: https://github.com/facebookresearch/faiss/issues/4260
```

Comment:
Not this one but
Line 49 below, `Qualitative parameters ...`

and last line, `[Error code descriptions source:](link)`
Just add a title using [Title](link)